contracts/scripts/lifebank.sh

```diff
@@ -136,7 +136,7 @@ create_community() {
         "community_asset":"0 LIFE",
         "description":"LifeBank development Instance",
         "logo":"https://raw.githubusercontent.com/eoscostarica/lifebank/master/docs/logos/2-OverWhite-lifebank-logo-v1-may25-2020-01.svg",
-        "maximum_supply":"1000000.0000 LIFE"
+        "maximum_supply":"1000000 EOS"
```

Comment:
`LIFE` not `EOS`

Reply:
I got your point. But we had some problems regarding to the precision and LIFE tokens. So we removed that and is now working